ui-meta/multinode-infra-appliance.yml

```diff
@@ -0,0 +1,183 @@
AAAAAOgq26cr0sLUerAmSd1cvSePSJsmAAAAAAAA6ErPTjXUmpNaD9gkqRublY5MUxMAAAAAAAB0rYlpo/TFdGtqPWiTpG5qdjosbZYAAAAAAACgZ2yezk/zUuuhmyS17pT0sjQ+AQAAAAAAQM/ZNR2X5qbWwzdJatGidG16YRqXAAAAAAAAoCfVsGvdND1dnVoP4iRpLFuQjko7JgAAAAAAAOgL66W90u2p9UBOksaiuu//x2mrNCkBAAAAAABAX3lK+li6OdWx2K0HdJI02i1OM9LuaWICAAAAAACAvrV6+nC6OLUe1EnSaFYLmy5J70sAAAAAAAAwEFZMT08/TwtT66GdJI20u9PxaVpaKQEAAAAAAMBAeUE6Ms1JrYd3kjSSjkhbpwkJAAAAAAAABtK4tEP6YrIQQFIvNT+dk16XAAAAAAAAgCXWSUenG9Pi1HqwJ0mP1j2prjF5fpqcAAAAAAAAgIdYLe2cTk1OA5DUrc1Nb00bpTrFhP9r795/vx8LOI4XbjoRpqR0PklNpXQupaOUNYVSa3OKxkqU0oGp1QxlbYVMLakIHZbEWCmHjFomQghRmUMOIcf7Xs9r05ZK43bf9/t7eDy25z/w/dz88rre1wUAAAAAAAD/w4LaoPathTX10CdJ/95l9blatQAAAAAAAID7aaP6Rl1bDgNImrI/1AG1VgEAAAAAAACLYYxt+9TZNfUAKGn+taj+VNvW6gUAAAAAAAA8CCvWc+qIurGmHgQlzY/GW/8H17iNxFv/AAAAAAAAsAQ9tz5SV9b4KnfqcVDS3G38f+ZL9YxaUAAAAAAAAMBS8Jjapi6oO2rqoVDS3GgcLLq8vlVrFgAAAAAAALCMvK9+WDeUGwEkPZhur/Nrl1q7AAAAAAAAgGVovMm9Wm1WR9TUA6Kk2dlt9clar1YoAAAAAAAAYCIr1br1zbq2ph4TJc2ezq3P1GMLAAAAAAAAmCHGjQBr1WfrtLq7ph4XJc287qoTasd6VAEAAAAAAAAz1Ir16jqprqpFNfXgKGlmdHWdWOvXODQEAAAAAAAAzALjSu831pF1XU09PEqarpvquHprrVkAAAAAAADALLNCrVvb1hU19Qgpadl3Y+1WL6gFBQAAAAAAAMxyq9am9aO6rKYeJSUtvcYX/2fWzvXEAgAAAAAAAOagp9VOdU7dXFMPlZKWXIvqmjqwxhMgvvgHAAAAAACAeeDZtVWdXA4CSLO/v9UBNd75X6kAAAAAAACAeeRhtUHtUlfV1AOmpMXrp7V9Pal89Q8AAAAAAADz3Kq1SR1dF9fUg6ak++72uqS+Wm8pAAAAAAAAgP+ydm1dZ9X1NfXQKene3VRH1Xa1SgEAAAAAAAD8X0+pTevIurqmHj2l+d544//AelcZ/gEAAAAAAIAHZLwlvm5tUWfWHTX1CCrNtxbWd+r99dRasQAAAAAAAAAW2/I1bgXYs06om2vqYVSaq11Tp9cetV4BAAAAAAAALHHL1Tp1UJ1R4yDAopp6MJXmQuO/p/Nq73pFrVAAAAAAAAAAS9W4hvyJtVkdUDfV1OOpNFsbX/wfU5vXeHbjoQUAAAAAAACwTC2ocRBgmzq8ri83Akj3r/HF/y9qi3peed8fAAAAAAAAmDFWrtfWEXV23VpTj6zSTOnOuqIOq+1qjfK1PwAAAAAAADCjrV4b16H1q3IQQPO5f9TFdXTtWo8uAAAAAAAAgFllXGk+nggYV5zvV3+s8RX01IOstCy6to6v8e//hfWI8sU/AAAAAAAAMKuNgwBPqDfV7nVh3VVTD7TS0ujSOry2rDH8LygAAAAAAACAOWn5GgcCxjvoh9Tv64aaeriVHmgL66I6qfaqN9RyBQAAAAAAADDvPKw2r/3rlBrPBIxRdephV7qvFtUt9bs6rraudcrwDwAAAAAAAJAxnq5VL6qd67D6a91dUw++0uiOGrdVfLo2rTH6r1IAAAAAAAAA3IeV61m1Se1eZ9VNNfUArPnZuJXi6NqhNqo1ytf+AAAAAAAAAIvpEbVh7VLj2vWz6/pyQ4CWVHfVNXVqfbs+UONGCgAAAAAAAACWksfXxrVPfb/Or+tqYU09Imt2Nd7zv7J+U9+rvWv9emQBAAAAAAAAsAw9vJ5Xr68P1yF1eY2vuacelzUzu7nOqYNrp3p5Pa1WKgAAAAAAAABmgFXq6fXm2rbGwPvLur2mHp01XeMr/zH6n1Lj38SW9Zp6aj2qAAAAAAAAAJglVq7n1zgUMJ4NOKF+XX+uW2vqgVpLptvqqjqjjq39avt6Zvm6HwAAAAAAAGAOWrs2qt3qoPp5nV1/KQcCZk/jZoera7zff1IdWnvUK2rNeug9vzcAAAAAAAAA88AYideo8ZX4K2uTGiPyIXVqXV5TD916yEMW1r/G/q/VJ2qz2rDG+/2r1XL3/mkBAAAAAAAAmO9WrzEqv7TeWDvWF+oHNZ4OuKWmHsTncuPN/nETw3l1/D1/+/EbjMMZr7rntxm/kcEfAAAAAAAAgAdl+RoD9Itr0xpfo3+5jqkTahwSOL/GkwI31Ph6fepRferGqD8OTlxTF9T4kv9n9eP6Sn2q3l2vqXEjw/gbAwAAAAAAAMAkHl5r1+tqy/pY7VdH1rF1cp1ZF9Z4WuDa+nvNhQMCY+C/rcaBhyvrovptnVbH1VE1hv69aqt6Qz27Vl2cPzQAAAAAAAAATGFcVT8OBzymnlzr1UtqHBR4S21R29bHa1x3f0h9t35S49DAOXVJXVVjYL+zlvagPw4ljC/2r6tL69w6vU6sI+rr9cUag/4O9Z56e40nE15eL6hn1OPrkeWLfgAAAAAAAADmvHFAYKVarR5XT6916vm1QY1r8V9fG9cY2d9Z47r8beqDtWvtXuMa/THIf772qXH7wGj//2jfGgcN9q49azxh8NH6UI0x/721eb2j3lZj1N+wXlbr13NqjPvjtoNxqGEcbjDwAwAAAACw1P0TkTDSy3qMdw4AAAAASUVORK5CYII=
```

Comment:
We could commit the image file to this repo, then link to it on GitHub?

Reply:
Can do and probably should to allow edits, but it did feel more 'lightweight' to contain it within the UI as a single, albeit very long, single line.